app/views/items/index.html.erb

```diff
@@ -15,7 +15,7 @@
   </section>
 
   <section class="disabled-items"><h4>Disabled Items</h4>
-  <% @items.disabled.each do |item| %>
+  <% @merchant.items.disabled.each do |item| %>
```

Comment:
I originally had it this way but second guessed myself that maybe it shouldn't be the view but perhaps in the controller. Glad you put it back!